includes/active-directory-cloud-sync-how-to-install.md

```diff
@@ -53,7 +53,7 @@ ms.custom: include file
 
 
 
- 13. Sign in with your Active Directory domain administrator account.  The domain administrator account shouldn't have password change requirements. In case the password expires or changes, you'll need to reconfigure the agent with the new credentials. This operation will add your on-premises directory. Select **OK**, then select **Next** to continue. 
+ 13. Sign in with your Active Directory domain administrator account.  The domain administrator account shouldn't have an expired password. In case the password is expired or changes during agent install, you'll need to reconfigure the agent with the new credentials. This operation will add your on-premises directory. Select **OK**, then select **Next** to continue. 
```

Comment:
```suggestion
 13. Sign in with your Active Directory domain administrator account.  The domain administrator account shouldn't have an expired password. In case the password is expired or changes during the agent installation, you'll need to reconfigure the agent with the new credentials. This operation will add your on-premises directory. Select **OK**, then select **Next** to continue. 
```